server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -100,6 +100,9 @@ grant {
   permission jdk.net.NetworkPermission "getOption.TCP_KEEPCOUNT";
   permission jdk.net.NetworkPermission "setOption.TCP_KEEPCOUNT";
 
+  permission java.lang.RuntimePermission "accessDeclaredMembers";
+  permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+  
```

Comment:
@dblock @saratvemulapalli @dbwiddis since we are removing jackson library from the native plugins and adding it in server. Need to add the above permissions. Looking for some advice if providing `accessDeclaredMembers` will bring any security concerns since we are enabling it for the server module? If not, then I guess we are good to go.

Reply:
^ We have these lines in feature/extensions as well

Reply:
Let's open an issue. I don't know what the impact of this change is.

Reply:
@ryanbogan can you take care of it?

Reply:
Already open: https://github.com/opensearch-project/OpenSearch/issues/5504